mm2src/coins/lp_coins.rs

```diff
@@ -3822,109 +3896,109 @@ where
 #[derive(Clone)]
 #[allow(clippy::large_enum_variant)]
 pub enum MmCoinEnum {
-    UtxoCoin(UtxoStandardCoin),
-    QtumCoin(QtumCoin),
-    Qrc20Coin(Qrc20Coin),
-    EthCoin(EthCoin),
-    ZCoin(ZCoin),
-    Bch(BchCoin),
-    SlpToken(SlpToken),
-    Tendermint(TendermintCoin),
-    TendermintToken(TendermintToken),
+    UtxoCoinVariant(UtxoStandardCoin),
+    QtumCoinVariant(QtumCoin),
+    Qrc20CoinVariant(Qrc20Coin),
+    EthCoinVariant(EthCoin),
+    ZCoinVariant(ZCoin),
+    BchVariant(BchCoin),
+    SlpTokenVariant(SlpToken),
+    TendermintVariant(TendermintCoin),
+    TendermintTokenVariant(TendermintToken),
     #[cfg(not(target_arch = "wasm32"))]
-    LightningCoin(LightningCoin),
+    LightningCoinVariant(LightningCoin),
     #[cfg(feature = "enable-sia")]
-    SiaCoin(SiaCoin),
+    SiaCoinVariant(SiaCoin),
     Solana(solana::SolanaCoin),
     SolanaToken(solana::SolanaToken),
     #[cfg(any(test, feature = "for-tests"))]
-    Test(TestCoin),
+    TestVariant(TestCoin),
 }
```

Comment:
Please note that the Variant suffix was added to every `MmCoinEnum` variant as a workaround for LLDB AST crash ref. https://github.com/llvm/llvm-project/issues/114068
please add a note about it so we can remove it when it's fixed, or alternatively change variants names (e.g. `Eth(EthCoin)` instead of `EthCoin(EthCoin)`). We should never use the below pattern until the issue is resolved.
```rust
enum OuterEnum {
    ValueType(ValueType),
}
```

Reply:
> please add a note about it so we can remove it when it's fixed

You should add a Note/TODO or

> or alternatively change variants names (e.g. Eth(EthCoin) instead of EthCoin(EthCoin)).